bundles/org.openhab.binding.shelly/src/main/java/org/openhab/binding/shelly/internal/util/ShellyUtils.java

```diff
@@ -55,6 +55,11 @@ public static String getString(@Nullable String value) {
         return value != null ? value : "";
     }
 
+    public static String getString(Exception e) {
```

Comment:
`getMessage` is a more appropriate name for this method.
```suggestion
    public static String getMessage(Exception e) {
```

Reply:
changed